oximeter/oximeter/schema/dendrite.toml

```diff
@@ -0,0 +1,58 @@
+format_version = 1
+
+[target]
+name = "dendrite"
+description = "Oxide switch management daemon"
+authz_scope = "fleet"
+versions = [
+    { version = 1, fields = [ "rack_id", "sled_model", "sled_revision", "sled_id", "sled_serial" ] },
+]
+
+[[metrics]]
+name = "sample_collection_duration"
+description = "Duration spent collecting all timeseries samples"
+units = "seconds"
+datum_type = "f64"
+versions = [
+    # Note: The sample collection time includes the time spent querying the
+    # switch for its statistics, which is why these fields are included.
+    # Dendrite may eventually report statistics about itself, or other aspects
+    # not related to the switch, so they belong here, not the target.
+    { added_in = 1, fields = [ "switch_model", "switch_revision", "switch_id", "switch_serial" ] }
+]
+
+[fields.rack_id]
+type = "uuid"
+description = "ID of the sled running Dendrite"
```

Comment:
This should be "ID of the rack..."

Who is the audience for this description?  If it's for Oxide internal use, this seems fine.  If it's ever going to be seen by customers, then the `running Dendrite` seems more confusing than helpful, as there is no reason they would know what that is.

Reply:
Good catch, thanks.

> Who is the audience for this description? 

As things are currently implemented, this can be read by anyone who can read fleet-level resources, which is operators and Oxide support staff. So yes, that includes customers. I agree that "Dendrite" isn't really something customers are aware of, and may be more confusing than helpful. Also, that applies to this whole timeseries, not just the field descriptions.

The visibility of the timeseries is currently controlled by the `authz_scope` key, which we could feasibly augment to include an "internal-only" variant. There are probably other ways to accomplish the same thing, such as a translation layer between the actual timeseries in the DB and those exposed via the Nexus API. That might be used to filter out this timeseries, for example, or just specific fields. But that's all vaporware at this point.

Given all that, what would you suggest as a better description?

Reply:
Maybe "ID of the rack containing the switch" and "ID (or serial, or whatever) of the sled managing the switch"?

Reply:
Updated in ef518a15c